45, Wild Wild Wet, Family, page: None, None, 2020-01-30 10:26:47.260813
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:49:32.409449
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:49:46.556358
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:50:04.359505
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:50:18.507880
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:50:33.848021
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:50:47.329998
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:51:00.528855
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:51:15.212724
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:51:33.006646
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:51:47.279664
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:52:05.773468
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:52:20.015425
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:52:33.760960
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:52:47.915571
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:53:05.488879
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:53:22.541885
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:53:40.872841
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:53:59.537060
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:54:13.040339
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:54:26.278942
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:54:44.922384
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:55:02.084886
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:55:19.859019
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:55:33.551822
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:55:47.210631
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:56:00.012269
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:56:13.663942
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:56:26.819974
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:56:40.372972
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:56:58.632465
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:57:16.555554
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:57:30.344801
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:57:43.823692
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:58:01.639142
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:58:15.734374
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:58:33.630882
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:58:51.138212
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:59:08.925693
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:59:26.542582
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:59:40.759012
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 10:59:56.067306
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:00:13.495818
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:00:31.567030
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:00:45.305429
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:00:59.514524
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:01:13.148283
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:01:30.828789
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:01:48.580275
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:02:02.178791
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:02:15.804977
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:02:29.231058
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:02:47.547158
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:03:00.500342
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:03:14.886874
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 165, in crawl_poi_by_trip_type
    '//div[@data-tracker="{}"]'.format(self.current_trip_type))
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@data-tracker="Family"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:03:34.000008
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 173, in crawl_poi_by_trip_type
    self.crawl_attributes()
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 253, in crawl_attributes
    '//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]')\
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 978, in find_element
    'value': value})['value']
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/ds/anaconda3/envs/crawl/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//div[@class="attractions-attraction-detail-about-card-AttractionDetailAboutCard__section--1_Efg"]"}
  (Session info: chrome=79.0.3945.130)


49, Suntec City, Family, page: None, None, 2020-01-30 11:03:46.185168
Traceback (most recent call last):
  File "/home/ds/git/STB_social_media_analytics/tripadvisor/tripadvisor_crawler.py", line 160, in crawl_poi_by_trip_type
    sleep(5)
KeyboardInterrupt

